scripts/get_hashsets.js

```diff
@@ -1,16 +1,18 @@
 "use strict";
 
-const AppConstants = require("./app-constants").init();
```

Comment:
While we're renaming the file... Should it be "get_hashsets.js" or "get-hashsets.js" (underscore vs hyphen)?
Not sure if we want to aim for file name consistency with the likes of "app-constants.js", etc.